events/README.md

```diff
@@ -0,0 +1,472 @@
+# Events
+
+## Events object
+
+```
+{
+  'name': string,
+  'description': string,
+  'room_id':string,
+  'template_id': string,
+  'enabled': boolean,
+  'lock': boolean,
+  'region': <'in' | 'us' | 'eu' | 'auto'>,
+  'peers': [
+    "<peer_id>",
+    "<peer_id>",
+    "<peer_id>"
+   ]
+  'created_by': {
+    ref: 'User',
+  },
+  'questions': [
+    "<question_id>",
+    "<question_id>",
+    "<question_id>"
+  ],
+  'comments: [
+    "<comment_id>",
+    "<comment_id>",
+    "<comment_id>",
+  ],
+  'status': <'active' | 'inactive'>,
+  'timestamp': {
+    'created_at': timestamp,
+    'updated_at': timestamp,
+  }
+}
+```
+
+## Peers object
+
+```
+{
+  'id': string,
+  'session_id': string,
+  'name': string,
+  'role': <'host' | 'moderator' | 'guest' | 'maven'>,
+  'user_id': string,
+  'joined_at': timestamp,
+  'left_at': timestamp,
+  'is_rds_user': boolean,
+  // extends user model
+}
+```
+
+## Questions object
+
+```
+{
+	'id': string,
+	'user_id': string,
+	'question': string,
+	'is_new': boolean, //default to true,
+	'session_id': string,
+	'timestamps': {
+      'created_at': timestamp,
+      'updated_at': timestamp,
+	}
+}
+```
+
+## Comments object
+
+```
+{
+	'id': string,
+	'comment': string,
+	'comment_by': string, //user id of the user who commented
+	'is_blocked': boolean,
+	'is_pinned': boolean,
+	'session_id': string,
+	'timestamps': {
+      'created_at': timestamp,
+      'updated_at': timestamp,
```

Comment:
same here

Reply:
Same here. It looks good on vscode. Not sure what's the issue here.

Reply:
Try to format with prettier. It should fix this issue hopefully.